config/routes.js

```diff
@@ -72,6 +72,12 @@ module.exports = function(app, passport, auth) {
     //Finish with setting up the userId param
     app.param('userId', users.user);
 
+    // Game routes
+    var game = require('../app/controllers/game');
+    app.get('/api/games', game.all);
```

Comment:
Tested this on deployment using cURL& works
![image](https://cloud.githubusercontent.com/assets/24469235/22427693/2ebfc8a0-e715-11e6-9525-28be467d1802.png)